src/main/java/dev/blaauwendraad/masker/json/KeyContainsMasker.java

```diff
@@ -141,21 +140,18 @@ private void visitObject(MaskingState maskingState, @CheckForNull KeyMaskingConf
         while (!AsciiCharacter.isCurlyBracketClose(maskingState.byteAtCurrentIndex())) {
             // In case target keys should be considered as allow list, we need to NOT mask certain keys
             int openingQuoteIndex = maskingState.currentIndex();
-            maskingState.incrementCurrentIndex(); // step over the JSON key opening quote
-            while (!currentByteIsUnescapedDoubleQuote(maskingState)) {
-                maskingState.incrementCurrentIndex();
-            }
+
+            skipStringValue(maskingState);
 
             int closingQuoteIndex = maskingState.currentIndex();
```

Comment:
At this point the current index is actually one step AFTER the closing quote, not on the closing quote, so this variable name is now wrong.

Reply:
You could:

`int afterClosingQuoteIndex = maskingState.currentIndex()` and keep -2 for the quotes you do later

or 

`int closingQuoteIndex = maskingState.currentIndex() - 1` and do minus 1 quote later

Reply:
Good catch 👍 Let's do the first suggestion. Or maybe we could also return the length of the key from skipStringValue method? Though we still need to remember the index before

Reply:
Eh, this is fine for now. I will send an update commit later